spec/models/chargeback_container_project_spec.rb

```diff
@@ -18,6 +18,11 @@
     temp = {:cb_rate => @cbr, :object => @ems}
     ChargebackRate.set_assignments(:compute, [temp])
 
+    cat = FactoryGirl.create(:classification, :description => "Environment", :name => "environment", :single_value => true, :show => true)
+    c = FactoryGirl.create(:classification, :name => "prod", :description => "Production", :parent_id => cat.id)
```

Comment:
Looking at factories/classification.rb:
- Use `FactoryGirl.create(:classification_tag` here.  `:classification` also sets `parent_id` to 0, which works same as you override it anyway, but :classification_tag describes your intent better.
- `:parent => cat`
- Can just call `FactoryGirl.create(:classification_department_with_tags)` and then use "/managed/department/finance" and a couple others.